resources/language/nl.ini

```diff
@@ -0,0 +1,489 @@
+language.code       = "nl"
+language.direction  = "ltr"
+language.isRTL = "false"
+language.dateformat = "d/m/Y"
```

Comment:
Slashes can be replaced with hyphens or periods if needed. (so d.n.Y or d-m-Y) is a valid format

Reply:
Good point! I will adjust it.